### DISCUSSION TITLE
Am thinking of buying T150,  exactly which FFB fuctions dont work?

### DISCUSSION THREAD

**Discussion by addeps3** [scarburato/t150_driver#28] [General]
Hi, I dont know which funtions "force feedback (partially) is missing" refers to, could someone please state them to me, because Im thinking of buying a T150 and am wondering which funtions I'd be missing out on more specifically?

**Comment by scarburato** [comment]
Please keep in mind Thrustmaster does not provide any official Linux support for their products and I simply can't guarantee the steering wheel will work, now or in the future, with your machine. Maybe it is wiser to consider a product with official Linux support, if they even exists on the market.

> I dont know which funtions "force feedback (partially) is missing" refers to

Basically some ffb effects may not be available with this driver even though they are present of the official MS Windows driver. At the moment I don't remember if I implemented all the effects or I left out something :P.

From my anecdotal experience I can state that at the moment the current implementation is sufficient to play most ffb games without major problems even though on some games you might have to tune down the ffb request rate as the driver does not use timers to queue requests to the wheel.

**Reply by addeps3** [reply]
> Please keep in mind Thrustmaster does not provide any official Linux support for their products and I simply can't guarantee the steering wheel will work, now or in the future, with your machine. Maybe it is wiser to consider a product with official Linux support, if they even exists on the market.
> 
> > I dont know which funtions "force feedback (partially) is missing" refers to
> 
> Basically some ffb effects may not be available with this driver even though they are present of the official MS Windows driver. At the moment I don't remember if I implemented all the effects or I left out something :P.
> 
> From my anecdotal experience I can state that at the moment the current implementation is sufficient to play most ffb games without major problems even though on some games you might have to tune down the ffb request rate as the driver does not use timers to queue requests to the wheel.

Understood. 
I agree, there are no guarantees. 

However, if we presume that I install and enable all drivers successfully (which Im confident, that I will be) is there any function of the wheel and racing experience that I would be missing out on because of the state of this driver? 

Essentially; would using and practicing with the wheel in Assetto Corsa/Competizione, rFactor 1 and 2 be the same, under Linux or Windows? 

Since a T150 is expensive, I it would be *so* helpful if someone could offer any information for potential buyers.

**Reply by albfan** [reply]
status for driver these days is that test like ffmvforce fftest or any check with jstest or similar just works

Even some config like gain or range can be configured with oversteer.

Don't think any critical feature is missing
